plugin_tests/dataone_register_test.py

```diff
@@ -28,17 +26,37 @@ def test_find_initial_pid(self):
         # Test that the regex is working for search.dataone urls
         pid = 'https://search.dataone.org/#view/urn:uuid:7ec733c4-aa63-405a-a58d-1d773a9025a9'
         res = find_initial_pid(pid)
-        assert res == 'urn:uuid:7ec733c4-aa63-405a-a58d-1d773a9025a9'
+        self.assertEqual (res,'urn:uuid:7ec733c4-aa63-405a-a58d-1d773a9025a9')
 
-        # Test that the regex is working for paths coming from the coordinating node
+        # Test that the regex is working for paths coming from the coordinating node v2
         pid = 'https://cn.dataone.org/cn/v2/object/urn:uuid:6f5533ab-6508-4ac7-82a3-1df88ed4580e'
         res = find_initial_pid(pid)
-        assert res == pid
+        self.assertEqual(res, 'urn:uuid:6f5533ab-6508-4ac7-82a3-1df88ed4580e')
+
+        # Test that the regex is working for paths coming from the coordinating node v1
+        pid = 'https://cn.dataone.org/cn/v1/object/urn:uuid:6f5533ab-6508-4ac7-82a3-1df88ed4580e'
+        res = find_initial_pid(pid)
+        self.assertEqual(res, 'urn:uuid:6f5533ab-6508-4ac7-82a3-1df88ed4580e')
+
+        # Test that the regex is working for paths with a d1
+        pid = 'https://cn.dataone.org/cn/d1/v2/object/urn:uuid:6f5533ab-6508-4ac7-82a3-1df88ed4580e'
```

Comment:
I don't think `/cn/d1/v2` is part of any DataONE URI. I usually see it as `/cn/v2/` w/o the `d1`. Have you seen this pattern before?